spring-backend/src/main/kotlin/com/example/sztoswro/member/MemberDAO.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.sztoswro.member
+
+import javax.persistence.*
+
+@Entity
+data class MemberDAO(
+        @Id
+        @GeneratedValue(strategy = GenerationType.SEQUENCE)
+        var id: Long,
+        var name: String,
+        var lastName: String,
+        var email: String,
+        val username: String,
+        var password: String,
+        @ElementCollection
+        var roles: MutableSet<String>
+//        @ManyToMany
+//        var organizations: MutableSet<OrganizationDAO>,
+
+) {
+        constructor(): this(1, "szymon", "nieszymon", "mozeszymon@mozenieszymon.com", "userszymon", "pass", mutableSetOf<String>())
+        constructor(mem: MemberDTO): this(mem.id, mem.name, mem.lastName, mem.email, mem.username, mem.password, mutableSetOf<String>())
```

Comment:
Tutaj skoro mamy dodawać autogenerowane ID to czemu musimy je dodawać w konstruktorze?

Reply:
Ta wartość nie może być pusta w konstruktorze, ale może być pusta przy dodawaniu membera (w jsonie) - wtedy będzie wygenerowana.